src/labels/CustomLabels.labels

```diff
@@ -2890,6 +2905,20 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Message warning user if totals don&apos;t match expected</shortDescription>
         <value>Require Expected Totals Match is selected but the Expected Count of Gifts and Expected Total Batch Amount fields are blank. Edit the batch details and either uncheck Require Expected Totals Match or add values in the related fields.</value>
     </labels>
+    <labels>
+        <fullName>bgeMassDryRun</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Text for mass dry run in BGE.</shortDescription>
+        <value>Mass Dry Run</value>
```

Comment:
Let's work with docs team and product to update the language here with their suggestions

Reply:
I have a meeting with @melissabarber on Thursday to discuss this!

Reply:
Yep, meeting about me directly editing this and updating the help text for this one: #4108 @ptewson-sfdo will we need to create another pull request for 4108 in order to make the update? I've used Github for project management, but I didn't ever touch the code during that project.

Reply:
@melissabarber - we will need to make pull requests to update the help text, but depending on your comfort level I'm happy for you to initiate the work or else to work the with the developer to handle the actual code change.

Reply:
@melissabarber - Happy to make the code changes as needed. Please feel free to ping me or schedule a chat any time :)

Reply:
@psadie & @ptewson-sfdo I added comments to the custom labels in the Files Changed tab. Let me know if you have any questions or if I missed anything. Thanks!